ui-src/src/app/test-suites/test-suites.component.ts

```diff
@@ -1,8 +1,171 @@
-import { Component, OnInit } from "@angular/core";
-
+import { Component, OnInit, Input, Output, EventEmitter } from "@angular/core";
+import { TestSuiteService } from "./test-suite.service";
+import { TestCaseService } from "../test-cases/test-case.service";
+import { ToastsManager } from "ng2-toastr/ng2-toastr";
+import { TestCasesSelectionComponent } from "../shared/test-cases-selection/test-cases-selection.component";
+const TEST_SUITE_PATH =
+  "C:/Users/A586754/go/src/github.com/xtracdev/automated-perf-test/config";
```

Comment:
Can we remove this, I think I put this there just to make easier for myself so I didn't have to keep copying and pasting the path